internal/cmd/identities.go

```diff
@@ -45,14 +45,16 @@ func newIdentitiesAddCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "add IDENTITY",
 		Short: "Create an identity.",
-		Long: `Create a machine identity with NAME or a user identity with EMAIL.
+		Long: `Create an identity.
 
-NAME must only contain alphanumeric characters ('a-z', 'A-Z', '0-9') or the
-special characters '-', '_', or '/' and has a maximum length of 256 characters.
-
-EMAIL must contain a valid email address in the form of "local@domain".
-		`,
+If an email is provided, a user is created; otherwise, a machine is created.
+A new user must change their one time password before further usage.`,
```

Comment:
The new description isn't entirely accurate since there are non-email that cannot be used as names, e.g. `a@`. I would suggest keeping the previous description

Reply:
For now we can error if there's a strange case (e.g. trying to name a user `a@`)

Reply:
reworded it as 'detects a valid email / username'